wallet/src/money.rs

```diff
@@ -18,6 +18,43 @@ use tuxedo_core::{
     verifier::Sr25519Signature,
 };
 
+use std::{thread::sleep, time::Duration};
+
+/// The default number of coins to be minted.
+pub const DEFAULT_NUM_OF_COINS: &str = "100";
+
+/// Create and send a transaction that mints the coins on the network
+pub async fn mint_coins(
+    client: &HttpClient,
+    args: MintCoinArgs,
+) -> anyhow::Result<()> {
+    log::info!("The args are:: {:?}", args);
+
+    let transaction = Transaction {
+        inputs: Vec::new(),
+        peeks: Vec::new(),
+        outputs: vec![(Coin::<0>::new(args.amount), OuterVerifier::Sr25519Signature(Sr25519Signature {
+            owner_pubkey: args.owner,
+        })).into()],
+        checker: OuterConstraintChecker::Money(MoneyConstraintChecker::Mint),
+    };
+
+
+    let spawn_hex = hex::encode(transaction.encode());
+    let params = rpc_params![spawn_hex];
+    let _spawn_response: Result<String, _> = client.request("author_submitExtrinsic", params).await;
+
+    let minted_coin_ref = OutputRef {
+        tx_hash: <BlakeTwo256 as Hash>::hash_of(&transaction.encode()),
+        index: 0,
+    };
+    sleep(Duration::from_secs(3));
+
+    let coin_from_storage = get_coin_from_storage(&minted_coin_ref, client).await?;
+    println!("Minted coin from storage = {:?}",coin_from_storage);
```

Comment:
This feels out of place to me. I think there is another command to check coin details in wallet storage and node storage. (double check that).

Reply:
Actually, I didn't find command for the user to get the coin details from the blockchain. ShowAllOutputs and ShowBalance get the data from the wallet db. So I added this.

Reply:
```
$ ./target/debug/tuxedo-template-wallet verify-coin --help
Verify that a particular coin exists.
Show its value and owner from both chain storage and the local database.

Usage: tuxedo-template-wallet verify-coin <OUTPUT_REF>

Arguments:
  <OUTPUT_REF>  A hex-encoded output reference

Options:
  -h, --help  Print help
```

Reply:
I fixed it as below:
Removed the "get_coin_from_storage()" after the transaction is sent. Instead, just print the coin reference, and later it can be verified by using the verify-coin command.